### PR TITLE
Ss gatk apply bqsr 4.1.2.0

### PR DESCRIPTION
**Pull request by shalabhsuman**
#32

### CODE REVIEW

**Comment by shalabhsuman** [comment]
@srinipr4 : I noticed we are utilizing `--tmp-dir .`. Just want to double-check if we should continue keeping it as`CWD (.)` or change it to anything else?

File: gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    inputBinding:
+      position: 0
+      prefix: '--disable-read-filter'
+  - id: disable_sequence_dictionary_validation
+    type: boolean?
+    inputBinding:
+      position: 0
+      prefix: '--disable-sequence-dictionary-validation'
+  - default: 'true'
+    'sbg:toolDefaultValue': 'true'
```
Comment (rhshah) [inline gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl]: - [ ] Remove sbg:toolDefaultValue

File: gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    type: int?
+  - id: memory_per_job
+    type: int?
+  - id: number_of_threads
+    type: int?
+outputs:
+  - id: output
+    type: File?
+    outputBinding:
+      glob: '$(inputs.input.basename.replace(''.bam'', '''')).recal.bam'
```
Comment (rhshah) [inline gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl]: - [ ] Please call the output file as `_bqsr.bam`

File: gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+    secondaryFiles:
+      - ^.bai
+label: gatk_apply_bqsr_4.1.2.0
+arguments:
+  - position: 0
+    prefix: '--output'
+    valueFrom: '$(inputs.input.basename.replace(''.bam'', '''')).recal.bam'
```
Comment (rhshah) [inline gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl]: - [ ] Please call the output file as `_bqsr.bam`

File: gatk_print_reads_4.1.2.0/gatk_ApplyBQSR.cwl [hunk 4]
```diff
@@ -0,0 +1,53 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
```
Comment (rhshah) [inline gatk_print_reads_4.1.2.0/gatk_ApplyBQSR.cwl]: - [ ] Remove sbg tag

File: gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+arguments:
+  - position: 0
+    prefix: '--java-options'
+    valueFrom: "${\n  if(inputs.memory_per_job && inputs.memory_overhead) {\n   \n    if(inputs.memory_per_job % 1000 == 0) {\n    \t\n      return \"-Xmx\" + (inputs.memory_per_job/1000).toString() + \"G\"\n    }\n    else {\n      \n      return \"-Xmx\" + Math.floor((inputs.memory_per_job/1000)).toString() + \"G\" \n    }\n  }\n  else if (inputs.memory_per_job && !inputs.memory_overhead){\n    \n    if(inputs.memory_per_job % 1000 == 0) {\n    \t\n      return \"-Xmx\" + (inputs.memory_per_job/1000).toString() + \"G\"\n    }\n    else {\n      \n      return \"-Xmx\" + Math.floor((inputs.memory_per_job/1000)).toString() + \"G\" \n    }\n  }\n  else if(!inputs.memory_per_job && inputs.memory_overhead){\n    \n    return \"-Xmx4G\"\n  }\n  else {\n    \n  \treturn \"-Xmx4G\"\n  }\n}"
+  - position: 2
+    prefix: '--output'
+    valueFrom: '$(inputs.input.basename.replace(''.bam'', '''') + ''_bqsr.bam'')'
+  - position: 2
+    prefix: '--tmp-dir'
+    valueFrom: .
```
Comment (ionox0) [inline gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl]: can this be an input?

File: gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl [hunk 6]
```diff
@@ -0,0 +1,247 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: gatk_apply_bqsr_4_1_2_0
+baseCommand:
```
Comment (rhshah) [inline gatk_ApplyBQSR_4.1.2.0/gatk_ApplyBQSR.cwl]: Can you also add the additional namespaces and the contributor tags

**Pull request merged**